policybot/pkg/storage/spanner/query.go

```diff
@@ -67,6 +67,44 @@ func (s store) QueryIssuesByRepo(context context.Context, orgID string, repoID s
 	return err
 }
 
+func (s store) QueryTestResultByName(context context.Context, testName string, cb func(*storage.TestResult) error) error {
+	iter := s.client.Single().Query(context, spanner.Statement{SQL: fmt.Sprintf("SELECT * FROM TestResults WHERE TestName = '%s'", testName)})
+	err := iter.Do(func(row *spanner.Row) error {
+		testResult := &storage.TestResult{}
+		if err := row.ToStruct(testResult); err != nil {
+			return err
+		}
+
+		if err := cb(testResult); err != nil {
```

Comment:
You don't need to be calling iter.Stop(). Check out the other query methods in this file for a slightly simplified code pattern to use here and in your other query method.